docs/en/Collaboration.md

```diff
@@ -0,0 +1,66 @@
+# Shared Workspaces with Kubeflow
+
+## What does Kubeflow do?
+
+Kubeflow runs your **workspaces**. You can have notebook servers (called Jupyter
+Servers), and in them you can create analyses in R and Python with interactive
+visuals. You can save & upload data, download it, and your team can work along side you.
+
+## Creating a shared workspace
+
+By default, everyone gets their own *personal* namespace, `firstname-lastname`. But if you want to create a namespace for a team, then there is a button to submit a request for a namespace on the portal. **Click the &#8942; menu on [the kubeflow section of the portal](https://portal.covid.cloud.statcan.ca/#kubeflow)**.
+
+!!! warning "The namespace cannot have special characters other than hypens"
+    The namespace name must only be lowercase letters `a-z` with dashes. Otherwise,
+    the namespace will not be created.
+
+**You will receive an email notification when the namespace is created.** You will then be able to manage the collaborators list through the kubeflow **Manage Contributors** tab
+
+![Contributors Menu](../images/kubeflow_contributors.png)
+
+![Contributors Menu](../images/kubeflow_manage_contributors.png)
+
+# Shared Programs
+
+Teams have two options (but you can combine both!):
+
+## Share a workspace in Kubeflow
+
+The advantage of sharing inside Kubeflow is that it's more free-form and it works better for `.ipynb` files (IPython Notebooks/Jupyter Notebooks). This method also lets you share a compute environment, so you can share resources very easily.
```

Comment:
Flesh out what gets shared when you share a namespace?  Notebook servers, persistent volumes in that workspace, minio storage for that namespace, .., so people quickly know what is affected